src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,5 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user: { name, email, username } }) => (
+  <a className="UserInfo" href={`mailto:${email || 'email unidentified'}`}>
+    {name || (username || 'name unidentified')}
```

Comment:
Similarly, in case the username or name is not provided, consider using a more descriptive message instead of 'name unidentified'.